### ISSUE TITLE
TensorRT8 in C++ ERROR

### ISSUE THREAD

**Issue by Huang-Yujiong** [Megvii-BaseDetection/YOLOX#1060]
i use the demo about demo>TensoRT>cpp  ;
when i build it  ;  that is a  error 

i want to know what shoul i do if i want to use TensoRT8 ? fix something?  

taankyou very much 

![0 3H3A}4LIBF8RE1H$Q8MM2](https://user-images.githubusercontent.com/70421213/148673642-317b9ca0-d480-43d6-9a18-6f214ff6cd09.png)

**Comment by qixuxiang** [comment]
You can try another modify version:https://github.com/shouxieai/tensorRT_Pro @Huang-Yujiong

**Comment by HyeRinYOO** [comment]
Same error for me.
Any update?

I tested it on jetson nano(2GB)
cuda10.2
tensorrt 8.0.1

**Comment by Huang-Yujiong** [comment]
yes ， it can be solve easily ； here is my change in the code .  
See  trt/include/logging.h ->>>  change some in this file 


![image](https://user-images.githubusercontent.com/70421213/150741411-53362f46-582c-48e6-9a9a-bbb3d83334f9.png)




Glad to help you bro !

**Comment by Huang-Yujiong** [comment]
> Same error for me. Any update?
> 
> I tested it on jetson nano(2GB) cuda10.2 tensorrt 8.0.1

i fix it , you can see the method  in the new comment . glad to help you

**Comment by HyeRinYOO** [comment]
![res](https://user-images.githubusercontent.com/47765743/151094182-245d4a92-6f3e-4024-8758-588ceed25e19.png)

It works for me too :)
Thanks!!
